UIComponents/UIComponents/Resources/Strings/Localizable.strings

```diff
@@ -7,6 +7,17 @@
 */
 // MARK: - PlaceHolder
 "username" = "Kullanıcı Adı";
-"email" = "E-mail Adresi";
+"email"    = "E-mail Adresi";
 "password" = "Şifre";
 
+// MARK: - Error
+"checkInformations" = "lütfen bilgilerinizi kontrol ediniz.";
```

Comment:
her ekran için yazılar aynı bile olsa farklı keyler ile tutulmalı, genel stringler kullanılmamalı